pubsub/api/Pubsub.Samples.Tests/PubsubFixture.cs

```diff
@@ -115,6 +115,32 @@ public Subscription CreateSubscription(string topicId, string subscriptionId)
         return subscription;
     }
 
+    public Subscription CreateExactlyOnceDeliverySubscription(string projectId, string topicId, string subscriptionId)
```

Comment:
Double check with the Pub/Sub team, we might need a sample for showing how to create an exactly once subscription, in which case, this method would become a sample.